pysmt/factory.py

```diff
@@ -441,6 +441,31 @@ def get_model(self, formula, solver_name=None, logic=None):
                 retval = solver.get_model()
             return retval
 
+    def get_implicant(self, formula, solver_name=None,
+                      logic=None):
+        mgr = self.environment.formula_manager
+        if logic == AUTO_LOGIC:
+            logic = get_logic(formula, self.environment)
+
+        with self.Solver(name=solver_name, logic=logic) \
+             as solver:
+            solver.add_assertion(formula)
+            check = solver.solve()
+            if not check:
+                return None
+            else:
+                model = solver.get_model()
+                atoms = formula.get_atoms()
+                res = []
+                for a in atoms:
+                    fv = a.get_free_variables()
+                    if any(v in model for v in fv):
+                        if solver.get_value(a).is_true():
```

Comment:
I don't have strong preferences. However, it seems like we never use get_py_value. Maybe we should just deprecate it?